### PR TITLE
Offline: implement `TableSchema` method

### PR DESCRIPTION
**Pull request by jinghancc**
This PR implements offline method `TableSchema` for all backends.

### CODE REVIEW

File: internal/database/offline/sqlite/store_test.go [hunk 1]
```diff
@@ -26,24 +29,43 @@ func prepareStore(t *testing.T) (context.Context, offline.Store) {
...
 }
 
-func destroyStore() {}
+func destroyStore() func() {
+	return func() {
+		if err := os.RemoveAll(DATABASE); err != nil {
+			panic(err)
+		}
+	}
+}
```
Comment (YikSanChan) [inline internal/database/offline/sqlite/store_test.go:38]:
```go
func destroyStore() {
  if err := os.RemoveAll(DATABASE); err != nil {
	panic(err)
  }
}
```

**Review by YikSanChan — APPROVED** [review]
lgtm

**Review by lianxmfor — APPROVED** [review]
lgtm

File: internal/database/offline/sqlite/store_test.go (hunk 1 above)
Comment (jinghancc) [reply internal/database/offline/sqlite/store_test.go]: done

**Pull request merged**